### PR TITLE
Easy model changes from 2023-05-09+ batch

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/risk.py [hunk 1]
```diff
@@ -145,20 +145,23 @@ def getModelDefs(self):
...
                     ('org:name', ('ou:name', {}), {
                         'ex': 'apt1',
-                        'doc': 'The reporting organization\'s name for the threat cluster.'}),
+                        'doc': "The reporting organization's name for the threat cluster."}),
 
                     ('org:names', ('array', {'type': 'ou:name', 'sorted': True, 'uniq': True}), {
                         'doc': 'An array of alternate names for the threat cluster, according to the reporting organization.'}),
 
+                    ('country', ('pol:country', {}), {
+                        'doc': "The reporting orgnaization's assessed country of origin of the threat cluster."}),
```
Comment (mbvertex) [inline synapse/models/risk.py:158]: typo - "orgnaization's"

**Review by mbvertex — APPROVED** [review]

**Review by mbvertex — APPROVED** [review]

File: synapse/models/inet.py [hunk 2]
```diff
@@ -1775,6 +1779,21 @@ def getModelDefs(self):
                             'doc': 'The contact information for the tunnel operator.'}),
                     )),
 
+                    ('inet:egress', {}, (
+
+                        ('host', ('it:host', {}), {
+                            'doc': 'The host that used the network egress.'}),
```
[the remark this answers is not in the document]
Comment (vEpiphyte) [reply synapse/models/inet.py:1785]: what about handling multiple egress ips for a given network?

File: synapse/models/risk.py [hunk 3]
```diff
@@ -569,6 +572,9 @@ def getModelDefs(self):
                         'ex': 'cno.breach',
                         'doc': 'A type for the compromise, as a taxonomy entry.'}),
 
+                    ('vector', ('risk:attack', {}), {
+                        'doc': 'The attack assessed to be the successful initial vector.'}),
```
Comment (vEpiphyte) [inline synapse/models/risk.py:576]: initial compromise vector

File: synapse/models/transport.py [hunk 4]
```diff
@@ -8,6 +8,15 @@ def getModelDefs(self):
...
                     'doc': 'A direction measured in degrees with 0.0 being true North.'}),
 
+                ('transport:land:vehicle', ('guid', {}), {
+                    'doc': 'An individual vehicle.'}),
+
+                ('transport:land:registration', ('guid', {}), {
+                    'doc': 'Registration issued to a contact for a land vehicle.'}),
+
+                ('transport:land:license', ('guid', {}), {
+                    'doc': 'A license to operate a land vehicle issued to a contact.'}),
```
Comment (vEpiphyte) [inline synapse/models/transport.py:18]: air licenses too?

File: synapse/models/inet.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (invisig0th) [reply synapse/models/inet.py:1785]: discussed IRL

File: synapse/models/transport.py (hunk 4 above)
Comment (invisig0th) [reply synapse/models/transport.py:18]: eventually, sure. out of scope for this PR.

File: synapse/lib/modelrev.py [hunk 5]
```diff
@@ -877,7 +883,10 @@ async def save():
                             continue
 
                         subcurv = sode['props'].get(subprop.name)
-                        if subcurv == subnorm: # pragma: no cover
+                        if subcurv is not None:
```
Comment (Cisphyx) [inline synapse/lib/modelrev.py:886]: do we want to compare the `(valu, stortype)` tuples here in case we need to update the stortype?

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**